.ci/jenkins/config/branch.yaml

```diff
@@ -49,13 +45,13 @@ maven:
       creds_id: unpacks-zip-on-qa-nexus
 cloud:
   image:
-    registry_credentials_nightly: nightly_kogito
-    registry_credentials_release: release_kogito
+    registry_credentials_nightly: rsynek_registry_token
+    registry_credentials_release: rsynek_registry_token
```

Comment:
@radtriste where do we use this key? I have seen only the "nightly" one be used as a default value of a job parameter.

Reply:
`registry_credentials_release` should be used in the release jobs

Reply:
Release pipeline will sent it via params: https://github.com/kiegroup/kogito-pipelines/blob/main/.ci/jenkins/Jenkinsfile.release#L617
It is set there: https://github.com/kiegroup/kogito-pipelines/blob/main/.ci/jenkins/dsl/jobs.groovy#L127